test/functional/cmdLineTests/runtimemxbeanTests/playlist.xml

```diff
@@ -23,14 +23,14 @@
 -->
 
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../TestConfig/playlist.xsd">
+	<include>variables.mk</include>
 	<test>
 		<testCaseName>cmdLineTester_getPid</testCaseName>
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
-		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) -DTESTDIR=$(Q)$(TEST_RESROOT)$(Q) -DRESJAR=$(CMDLINETESTER_RESJAR) -DEXE='$(JAVA_COMMAND) $(JVM_OPTIONS)' -DTESTSJARPATH=$(Q)$(TEST_RESROOT)$(D)runtimemxbeanTests.jar$(Q) -jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)getPidTest.xml$(Q) \
-		-explainExcludes -xids all,$(PLATFORM),$(VARIATION) -xlist $(Q)$(TEST_RESROOT)$(D)exclude.xml$(Q) -nonZeroExitWhenError; \
-		$(TEST_STATUS)</command>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) -DTESTDIR=$(Q)$(JVM_TEST_ROOT)$(GETPID_PATH)functional$(GETPID_PATH)cmdLineTests$(GETPID_PATH)runtimemxbeanTests$(Q) -DRESJAR=$(GETPID_CMDLINETESTER_RESJAR) -DEXE='$(JAVA_COMMAND) $(JVM_OPTIONS)' -DTESTSJARPATH=$(Q)$(JVM_TEST_ROOT)$(GETPID_PATH)functional$(GETPID_PATH)cmdLineTests$(GETPID_PATH)runtimemxbeanTests$(GETPID_PATH)runtimemxbeanTests.jar$(Q) -jar $(GETPID_CMDLINETESTER_JAR) -config $(Q)$(JVM_TEST_ROOT)$(GETPID_PATH)functional$(GETPID_PATH)cmdLineTests$(GETPID_PATH)runtimemxbeanTests$(GETPID_PATH)getPidTest.xml$(Q) -explainExcludes -nonZeroExitWhenError; \
+ 		$(TEST_STATUS)</command>
```

Comment:
- remove `-DRESJAR=$(GETPID_CMDLINETESTER_RESJAR)` in playlist.xml
- remove GETPID_CMDLINETESTER_RESJAR in variables.mk
- can we name `GETPID_PATH` some more generic?
- make the following `$(JVM_TEST_ROOT)$(GETPID_PATH)functional$(GETPID_PATH)cmdLineTests$(GETPID_PATH)runtimemxbeanTests$(GETPID_PATH)` to be a variable in mk